### PR TITLE
Make method less complex, add Code Climate badge

### PR DESCRIPTION
**Pull request by mrb**
Hey there! I ran the `ember-rails` code through Code Climate when I had some spare time to do some OSS work, and the results are great! I made one small fix that it uncovered, to help make a method a bit more readable and to pull some code out that you may eventually want to test separately. I also added the Code Climate badge to your README. Thanks for the awesome lib!

### CODE REVIEW

File: lib/generators/ember/bootstrap_generator.rb [hunk 1]
```diff
@@ -68,6 +60,17 @@ def inject_into_application_file(safe_extension)
           template application_file, full_path
         end
       end
+
+      def get_options_from_path(path)
```
Comment (tricknotes) [inline lib/generators/ember/bootstrap_generator.rb]: Probably `path` -> `contents` ?

Comment (mrb) [inline lib/generators/ember/bootstrap_generator.rb]: @tricknotes Do you mean that the method should be named `get_options_from_contents`? Or that the parameter should be named `contents`?

Comment (tricknotes) [inline lib/generators/ember/bootstrap_generator.rb]: Yes, both of them.
It seems this method takes as a file contents as an argument, not a file path.

**Comment by mrb** [comment]
@tricknotes Your comment has been addressed.

**Comment by tricknotes** [comment]
Thanks @mrb :dizzy:

**Pull request merged**